### ISSUE TITLE
$feed_title in posts does not match content of title field in feed database

### ISSUE THREAD

**Issue by tux93** [maubot/rss#49]
My use case is the following: I have subscribed to a feed but want to customize the title shown when new posts are sent.

To do this I tried to modify the 'title' field in the 'feed' database, but new posts are still sent with the title that was set when subscribing, my notification_template looks like this: `New post from $feed_title:<br/>$link`

**Comment by tux93** [comment]
> You can just type a custom title in the template itself rather than using the variable

True, but is the expectation that the variable matches the database wrong?

**Comment by tulir** [comment]
Yes, the database is just a cache used for `!rss subscriptions` and other cases where the bot doesn't re-fetch the feed. When broadcasting new entries, the bot has just re-fetched the feed, so it has the latest feed info and has no reason to use cached data.

It should probably also update the database if it doesn't match though 🤔
